packages/contracts/.gitignore

```diff
@@ -1,5 +1,7 @@
 node_modules
 .env
+temp
```

Comment:
Temp, from where is it coming?

Reply:
It's for me to do deployments scripts! Take out 001 and put in tmp before I want to upgrade one contract